astropy/utils/data_info.py

```diff
@@ -320,13 +320,13 @@ def _parent(self):
 
         if parent is None:
             raise AttributeError("""\
-failed access "info" attribute on a temporary object.
+Failed to access "info" attribute on a temporary object.
```

Comment:
I don't think capitalizing is necessary here. For one thing, if anyone is doing a match on the test, it would probably be on this initial section. Keeping it lower cased assists in backward compatibility.

Reply:
I am adding a whole new word there, so I don't think changing the capitalization is too egregious. But I can undo the capitalization change if more people think it would be a good idea.

Reply:
Python's error messages are not capitalized, which makes some sense, since the string one adds will always come after a colon. To match this behaviour, some time ago we made an effort to remove all capitalization (surely, still some remaining...).

p.s. E.g., `1 + list()` gives:
```
TypeError: unsupported operand type(s) for +: 'int' and 'list'
```

Reply:
The capitalization change should be undone, but adding "to" still looks like an improvement.

Reply:
Indeed, keep the "to"

Reply:
Searching on GitHub, I don't see any code (except for forks of astropy) that checks for ``failed access "info"``.
My search term was ``"failed access \"info\" "``.

Reply:
Ok. Found one https://github.com/InterImm/poliastro-astropy-lambda-layer/blob/7587eceb68ab76281f3dede9ce170bfd76aa4474/poliastro/python/astropy/table/tests/test_info.py
But last active 2 years ago.

Reply:
I have always thought matching exact error messages was a bit bad form - I don't think we should let that stop us from correcting grammatical mistakes.